mimicpy/parsers/top_reader.py

```diff
@@ -45,9 +45,14 @@ def parseBlocktillSection(file, *sections):
         
     return itp_txt
 
-def cleanText(txt):
+def cleanText(txt, keep_hash=False):
```

Comment:
Best solution: give a string of characters to be cleared as argument.

Reply:
Good idea